landing/src/input.css

```
@@ -0,0 +1,31 @@
+/* landing/src/input.css */
+@import "tailwindcss";
+@source "../";
```

Stylelint (17.6.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)